ovirt-imageio.spec.in

```diff
@@ -145,6 +184,56 @@ if systemctl is-active ovirt-imageio-daemon.service >/dev/null; then
     systemctl stop ovirt-imageio-daemon.service
 fi
 
+%if %{with_python38}
+%define python3_pkgversion 38
+%define __python3 /usr/bin/python3.8
+
+%package -n python38-%{name}-common
+Summary:   oVirt imageio common resources
+
+# NOTE: keep in sync with automation/build-artifacts.packages
+BuildRequires: gcc
+BuildRequires: python38-devel
+
```

Comment:
Any reason for the blank line here?

Reply:
Isn't more readable to have a blank line between BuildRequires and Requires?

Reply:
Right, I missed it.